### DISCUSSION TITLE
Restrictions to export data [Jul 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6048] [Discord Threads]
> 🙋‍♂️ **kots**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not possible. Our philosophy, if user can see data - should be ok to download as well. What's your take on this?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1115538480399913011)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
